content/articles/most-recent-article.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Most recent article
+date: 2020-03-02:08:38.444Z
```

Comment:
Invalid date - missing hour component. 

Causes the following error
`Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments: 
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: 2020-03-02:08:38.444Z, _f: undefined, _strict: undefined, _locale: [object Object]
Error
    at Function.createFromInputFallback (./dev/census-worth-self-help/site/public/render-page.js:32865:98)`